src/state/account.ts

```diff
@@ -372,10 +400,56 @@ export const accountSlice = createSlice({
     ) => {
       state.localCancelOrders = state.localCancelOrders.map((order) =>
         order.orderId === action.payload.orderId
-          ? { ...order, errorParams: action.payload.errorParams }
+          ? {
+              ...order,
+              errorParams: action.payload.errorParams,
+            }
           : order
       );
     },
+    cancelAllSubmitted: (
+      state,
+      action: PayloadAction<{ marketId?: string; orderIds: string[] }>
```

Comment:
nit, can we throw a comment explaining that not passing in a market id globally cancels all orders and passing in one only cancels orders for that market? I have a slight thought that it might also be clearer if we split it into `cancelAllSubmitted` and `cancelAllForMarketSubmitted` but I can also see that being a lil verbose